test/SdkTests/SdkTests.csproj

```diff
@@ -46,8 +46,8 @@
     <Reference Include="BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942">
       <HintPath>..\packages\BouncyCastle.1.8.9\lib\BouncyCastle.Crypto.dll</HintPath>
     </Reference>
-    <Reference Include="DocuSign.eSign">
-      <HintPath>..\..\sdk\src\DocuSign.eSign\bin\Debug\net452\DocuSign.eSign.dll</HintPath>
```

Comment:
I think we should revert this file. If we're going to provide tests, we should provide them in a way that will use their locally built SDK client. I think this not being honored during releases is related to .gitignore file.